docs/pages/setup/guides/terraform-provider.mdx

```diff
@@ -140,6 +141,7 @@ terraform {
 
 provider "teleport" {
   # Update addr to point to Teleport Auth/Proxy
+  # addr               = "auth.example.com:3025"
```

Comment:
If I use the proxy:

```
│ Error: all auth methods failed
│ 	context deadline exceeded, context deadline exceeded
│ 
│   with provider["gravitational.com/teleport/teleport"],
│   on main.tf line 10, in provider "teleport":
│   10: provider "teleport" {
```

Using the `auth` port `3025` works magnificently.

I figure this is the result of a configuration choice on my end (it also mentions something like this in the README). I've added the `auth` port and example as a comment.